.github/meta/update_labels_and_milestones.py

```diff
@@ -0,0 +1,196 @@
+from github import Github
+
+
+repos = [
+    "OpenSlides",
+    "openslides-auth-service",
+    "openslides-autoupdate-service",
+    "openslides-backend",
+    "openslides-client",
+    "openslides-datastore-service",
+    "openslides-icc-service",
+    "openslides-manage-service",
+    "openslides-media-service",
+    "openslides-vote-service",
+    "vote-decrypt",
+]
+
+labels = {
+    "general": [
+        {
+            "name": "blocker",
+            "color": "fbca04",
+        },
+        {
+            "name": "bug",
+            "color": "b60205",
+        },
+        {
+            "name": "clean up",
+            "color": "c5def5",
+        },
+        {
+            "name": "critical",
+            "color": "000000",
+            "description": "Highest priority: This has to be done first."
+        },
+        {
+            "name": "dependencies",
+            "color": "c5def5",
+        },
+        {
+            "name": "documentation",
+            "color": "c5def5",
+        },
+        {
+            "name": "enhancement",
+            "color": "006b75",
+            "description": "General enhancement which is neither bug nor feature"
+        },
+        {
+            "name": "feature",
+            "color": "0e8a16",
+        },
+        {
+            "name": "good first issue",
+            "color": "5319e7",
+        },
+        {
+            "name": "high",
+            "color": "d93f0b",
+        },
+        {
+            "name": "low",
+            "color": "fbca04",
+        },
+        {
+            "name": "needs discussion",
+            "color": "c5def5",
+        },
+        {
+            "name": "needs info",
+            "color": "c5def5",
+        },
+        {
+            "name": "needs investigation",
+            "color": "e99695",
+        },
+        {
+            "name": "performance",
+            "color": "006b75",
+        },
+        {
+            "name": "waiting",
+            "color": "cccccc",
+            "description": "Waiting for some other PR/feature; more details in comments"
+        },
+    ],
+    "OpenSlides": [
+        {
+            "name": "help wanted",
```

Comment:
Proposal: Move `help wanted` to general.

Reply:
If someone needs help on an issue, he or she should communicate this with the team using internal channels and a solution will be found. I don't see the value of such a label except in the main repo, where users will ask for help without anyone specific to ask and can label their issue accordingly.

Reply:
Ok